SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+## Supported Versions
+
+The OpenIndiana project commits to providing fixes or mitigations to each
+and every security vulnerabilities in software we provide. List of source
+code repositories of software we develop is to be found at https://github.com/OpenIndiana/.
```

Comment:
Should we use direct link to github or https://docs.openindiana.org/dev/building-openindiana/#source-repositories (as last link contains also software description)?